examples/generate_example.py

```diff
@@ -2,25 +2,56 @@
 """
 Generate example STAC from test data
 """
+from pathlib import Path
+
 import pystac
 
-from stactools.sentinel1.rtc import stac
+from stactools.sentinel1.grd import Format
+from stactools.sentinel1.grd import stac as grd
+from stactools.sentinel1.rtc import stac as rtc
+
+# GRD generate examples
+root = Path(__file__).parents[1]
+examples = root / "examples" / "grd"
+grd_data = root / "tests" / "data-files" / "grd"
+
+grd_collection = grd.create_collection(str(examples))
+
+item1 = grd.create_item(
+    str(
+        grd_data / "S1A_EW_GRDM_1SDH_20221130T014342_20221130T014446_046117_058549_BB15"
+    ),
+    archive_format=Format.COG,
+)
+item2 = grd.create_item(
```

Comment:
While it is important to `test` both COG and SAFE, do the item records come out any different? Just not sure it's necessary, at the same time, it's one json file so not a big deal.

Reply:
It was more, just running generate_examples on everything in the repository. I saw that the cop-dem stactools did a glob on all the tiles

Reply:
ah, the glob was only there as a copy from the NLCD example that actually had multiple tiles